web-terminal/instance/overlays/enhanced/web-terminal-tooling.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: workspace.devfile.io/v1alpha2
+kind: DevWorkspaceTemplate
+metadata:
+  annotations:
+    controller.devfile.io/allow-import-from: '*'
+    web-terminal.redhat.com/unmanaged-state: 'true'
+  name: web-terminal-tooling
+  namespace: openshift-operators
+  labels:
+    console.openshift.io/terminal: 'true'
+spec:
+  components:
+    - container:
+        env:
+          - name: YOLO_URL
+            value: https://raw.githubusercontent.com/codekow/demo-ai-gitops-catalog/main/scripts/library/term.sh
```

Comment:
Instead of curling this URL what about setting up the script with a CM generator in kustomize and mounting it into the image instead?

Reply:
That's a fun idea. With this example I'm still pulling external dependencies in real time. So you will still be git cloning and other things. The initial effort was to not modify the terminal container and just make it work as is. An actual build config to produce a container is an alternative we could explore. There are many ways to do this, this being the path of least resistance and effort :)

Reply:
I'm good with this @codekow .  Waiting on @strangiato to approve.